pkg/apiserver/metrics/metrics.go

```diff
@@ -42,7 +42,7 @@ var (
 			// Use buckets ranging from 125 ms to 8 seconds.
 			Buckets: prometheus.ExponentialBuckets(125000, 2.0, 7),
 		},
-		[]string{"verb", "resource"},
+		[]string{"verb", "resource", "client"},
```

Comment:
Note that another approach might be:
- leaving the existing metric as is
- add another metric that does just an "entry per client" (without separation for verb and resource)
Although, there still may be a lot of clients.